File: Plugins/Aspose_Slides_Java_for_PHP/src/aspose/slides/WorkingWithSlidesInPresentation/AccessSlides.php
<?php

namespace Aspose\Slides\WorkingWithSlidesInPresentation;

use com\aspose\slides\Presentation as Presentation;
use com\aspose\slides\SaveFormat as SaveFormat;

class AccessSlides {

    public static function run($dataDir=null)
    {

        # Using Slides Collection to Access Slide by Index
        AccessSlides::get_slide_by_index($dataDir);

        # Using Slides Collection to Access Slide by ID
        AccessSlides::get_slide_by_id($dataDir);
    }

    public static function get_slide_by_index($dataDir=null)
    {

        # Instantiate Presentation class that represents the presentation file
        $pres = new Presentation($dataDir . 'Aspose.pptx');

        # Accessing a slide using its slide index
        $slide = $pres->getSlides()->get_Item(0);

        print "Slide: " . $slide . PHP_EOL;

    }

    public static function get_slide_by_id($dataDir=null)
    {

        # Instantiate Presentation class that represents the presentation file
        $pres = new Presentation($dataDir . 'Aspose.pptx');

        # Getting Slide ID
        $id = $pres->getSlides()->get_Item(0)->getSlideId();

        # Accessing Slide by ID
        $slide = $pres->getSlideById($id);

        print "Slide: " . $slide . PHP_EOL;

    }

}

File: Plugins/Aspose-Slides-Java-for-Jython/asposeslides/WorkingWithSlidesInPresentation/AccessSlides.py
from asposeslides import Settings
from com.aspose.slides import Presentation
from com.aspose.slides import SaveFormat

class AccessSlides:

    def __init__(self):
        
        # Formatting the Lines of Shapes
        self.get_slide_by_index()

        # Formatting the Join Styles
        self.get_slide_by_id()
        
    def get_slide_by_index(dataDir):
        
        dataDir = Settings.dataDir + 'WorkingWithSlidesInPresentation/AccessSlides/'
        
        # Instantiate Presentation class that represents the presentation file
        pres = Presentation(dataDir + 'Aspose.pptx')

        # Accessing a slide using its slide index
        slide = pres.getSlides().get_Item(0)

        print "Slide: "
        print slide
        
    def get_slide_by_id(dataDir):
        
        dataDir = Settings.dataDir + 'WorkingWithSlidesInPresentation/AccessSlides/'
        
        # Instantiate Presentation class that represents the presentation file
        pres = Presentation(dataDir + 'Aspose.pptx')

        # Getting Slide ID
        id = pres.getSlides().get_Item(0).getSlideId()

        # Accessing Slide by ID
        slide = pres.getSlideById(id)

        print "Slide: "
        print slide

if __name__ == '__main__':        
    AccessSlides()